app/models/miq_schedule_worker/jobs.rb

```diff
@@ -62,7 +62,7 @@ def job_check_for_evm_snapshots(job_not_found_delay)
 
   def job_proxy_dispatcher_dispatch
     if JobProxyDispatcher.waiting?
-      queue_work_on_each_zone(:class_name => "JobProxyDispatcher", :method_name => "dispatch", :task_id => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY, :role => "smartstate")
+      queue_work_on_each_zone(:class_name => "JobProxyDispatcher", :method_name => "dispatch", :tracking_label => "job_dispatcher", :priority => MiqQueue::HIGH_PRIORITY, :role => "smartstate")
```

Comment:
I think this is intentional to avoid multiple job_proxy_dispatchers from running simultaneously.